modules/instrument_list/php/instrument_list_controlpanel.class.inc

```diff
@@ -117,57 +117,7 @@ class Instrument_List_ControlPanel extends \TimePoint
             && in_array($currentStage, array('Visit', 'Screening'))
             && $currentStatus != 'In Progress'
         ) {
-
```

Comment:
Is suppression of the following lines (120-170) really part of this PR?

Reply:
Yes there was something about calling singleton on `TimePoint_Flag`....  a module that does not exist.

I think this section of the code is very old and somehow escaped cleanup.